.github/workflows/cd-pipeline.yml

```diff
@@ -59,6 +60,21 @@ jobs:
         with:
           script: |
               core.setFailed('${{ steps.get-chart.outputs.error }}')
+      - id: push-tag
```

Comment:
Not sure if this is the best place to push the tag, at this point the chart is not yet published and the repository is not updated with the new version. We can reach a situation where the tag is created and the chart is not published. We could leave this job as read-only and create  a new job depending on the `update-index` to push the tag if everything went well.

Reply:
AFAIK, several in several repositories release jobs get triggered when a tag is set, right?